frame/fee-market/src/lib.rs

```diff
@@ -105,38 +103,34 @@ pub mod pallet {
 	)]
 	pub enum Event<T: Config> {
 		/// Relayer enrollment
-		EnrollAndLockCollateral(T::AccountId, RingBalance<T>, Fee<T>),
-		/// Update relayer locked collateral
-		UpdateLockedCollateral(T::AccountId, RingBalance<T>),
-		/// Update relayer fee
-		UpdateRelayFee(T::AccountId, Fee<T>),
+		Enroll(T::AccountId, RingBalance<T>, Fee<T>),
+		/// Update relayer
+		UpdateRelayer(T::AccountId, Option<RingBalance<T>>, Option<Fee<T>>),
```

Comment:
suggest use two different Event to deposit Collateral update event and Fee update event